docs/ops/config.md

```diff
@@ -100,9 +100,9 @@ The default fraction for managed memory can be adjusted using the taskmanager.me
 
 {% include generated/security_configuration.html %}
 
-### Network Environment
+### Netty Shuffle Environment
 
-{% include generated/network_environment_configuration.html %}
```

Comment:
this file should also be deleted, I don't think this is used anywhere.

Reply:
fixed